tests/CoreCLR/runtest/dir.props

```diff
@@ -1,17 +1,14 @@
 <Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
```

Comment:
This is needed because the `tests/CoreCLR/runtests/tests.targets` file uses `xunit.runner.msbuild` which is Desktop only.

The roslyn toolset we are restoring requires net472 for the desktop toolset, which is not installed on all the Jenkins build-machines.